.github/workflows/workflow.yaml

```
@@ -39,16 +45,27 @@ on:
+    strategy:
+      matrix:
+        include: ${{ fromJSON(needs.setup.outputs.matrix) }}
+    runs-on: ${{ matrix.runs-on }}
             
     permissions:
       packages: write
       contents: write
 
     steps:
 
+    - uses: runs-on/action@v1
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 6 but found 4

(indentation)